src/parser/priest/shadow/modules/shadowlands/conduits/DissonantEchoes.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events, { DamageEvent } from 'parser/core/Events';
+import EventHistory from 'parser/shared/modules/EventHistory';
+import { Options } from 'parser/core/Module';
+import { t, Trans } from '@lingui/macro';
+
+import Statistic from 'interface/statistics/Statistic';
+import ItemDamageDone from 'interface/ItemDamageDone';
+import STATISTIC_ORDER from 'interface/others/STATISTIC_ORDER';
+import STATISTIC_CATEGORY from 'interface/others/STATISTIC_CATEGORY';
+import { When, ThresholdStyle } from 'parser/core/ParseResults';
+import ConduitSpellText from 'interface/statistics/components/ConduitSpellText';
+
+import { DISSONANT_ECHOES_DAMAGE_INCREASE } from 'parser/priest/shadow/constants';
+
+class DissonantEchoes extends Analyzer {
+  static dependencies = {
+    eventHistory: EventHistory,
+  };
+  protected eventHistory!: EventHistory;
+
+  conduitRank = 0;
+  damage = 0;
+  procsGained = 0;
+  procsUsed = 0;
+
+  get procsWasted() {
+    return this.procsGained - this.procsUsed;
+  }
+
+  get procsWastedPercentage() {
+    return this.procsWasted / this.procsGained;
+  }
+
+  constructor(options: Options) {
+    super(options);
+
+    this.conduitRank = this.selectedCombatant.conduitRankBySpellID(SPELLS.DISSONANT_ECHOES.id);
+    if (!this.conduitRank) {
```

Comment:
Actually, from the looks of it, conduitRank doesnt come out false if the conduit is not equipped. So this wont exit (Or at least it isnt on my mage ones that i tried it on)

Reply:
would need to use this.selectedCombatant.hasConduitBySpellID, or i suppose check to see if the rank is > 0

Reply:
I am not seeing the same result on my end. Threw some console logs before the if, in it, and after and they're all only rendering when appropriate.

Reply:
damn not sure what is up with my stuff lately. But i went to change my conduits cause i didnt know ConduitSpellText was a thing, and tried eliminating the this.active and just using conduitRank and it was showing conduits that i didnt have equipped.

And now it isnt ... so idk, my vscode is haunted or something